geany.appdata.xml.in

```diff
@@ -0,0 +1,43 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<component>
+  <id type="desktop">geany.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <_name>Geany</_name>
+  <project_license>GPL-2.0+</project_license>
+  <_summary>A fast and lightweight IDE</_summary>
+  <url type="homepage">http://geany.org/</url>
+  <description>
+    <_p>Geany was developed to provide a small and fast IDE, which has only a
+few dependencies on other packages. Another goal was to be as independent
+as possible from a specific Desktop Environment like KDE or GNOME.</_p>
+
+    <_p>Geany includes the following features:</_p>
+    <ul>
+      <_li>Syntax highlighting</_li>
+      <_li>Code completion</_li>
+      <_li>Auto completion of often used constructs like if, for and while</_li>
+      <_li>Auto completion of XML and HTML tags</_li>
+      <_li>Call tips</_li>
+      <_li>Code folding</_li>
+      <_li>Many supported filetypes like C, Java, PHP, HTML, Python, Perl, Pascal</_li>
+      <_li>Symbol lists</_li>
+      <_li>Embedded terminal emulation</_li>
+      <_li>Extensibility through plugins.</_li>
+    </ul>
+  </description>
+  <screenshots>
+<!-- Screenshots are recommended to be 16:9 ratio but the ones here from the main website are not. Please
+     change the links to point to appropriate 16:9 sized screenshots. -->
+    <screenshot type="default">
+      <image height="808" width="942">http://www.geany.org/uploads/Gallery/geany_main.png</image>
+    </screenshot>
+    <screenshot>
+      <image height="808" width="942">http://www.geany.org/uploads/Gallery/geany_build.png</image>
+    </screenshot>
+    <screenshot>
+      <image height="808" width="942">http://www.geany.org/uploads/Gallery/geany_vte.png</image>
+    </screenshot>
+  </screenshots>
+  <update_contact>https://github.com/geany/geany</update_contact>
+  <translation type="gettext">@GETTEXT_PACKAGE@</translation>
```

Comment:
That won't work just like that, it need to be replaced.  Something like that would work:
```diff
diff --git a/Makefile.am b/Makefile.am
index 1e65e44..351afb8 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -17,7 +17,7 @@ WIN32_BUILD_FILES = \
 EXTRA_DIST = \
 	autogen.sh \
 	scripts/gen-api-gtkdoc.py \
-	geany.appdata.xml.in \
+	geany.appdata.xml.in.in \
 	geany.desktop.in \
 	geany.pc.in \
 	geany.spec \
diff --git a/configure.ac b/configure.ac
index d921e95..6407bda 100644
--- a/configure.ac
+++ b/configure.ac
@@ -162,6 +162,7 @@ AC_CONFIG_FILES([
 		geany.spec
 		geany.pc
 		geany.nsi
+		geany.appdata.xml.in
 		doc/Doxyfile
 		tests/Makefile
 		tests/ctags/Makefile
diff --git a/geany.appdata.xml.in b/geany.appdata.xml.in.in
similarity index 100%
rename from geany.appdata.xml.in
rename to geany.appdata.xml.in.in
```